exercises/concept/savings-account/savings_account_test.rb

```diff
@@ -78,8 +78,8 @@ def test_annual_balance_update_for_small_negative_start_balance
     assert_equal(-0.12_695_199, SavingsAccount.annual_balance_update(-0.123))
   end
 
-  def test_annual_balance_update_for_avarage_negative_start_balance
-    assert_equal 1_016.21, SavingsAccount.annual_balance_update(1_000.0)
+  def test_annual_balance_update_for_average_negative_start_balance
+    assert_equal(-1_032.13, SavingsAccount.annual_balance_update(-1_000.0))
```

Comment:
This should be by delta, rather than an exact amount, to allow for various approaches (and perhaps even different machine architectures).  See #1199

Reply:
Yes please, unless it's always rounded to 2 decimals.